plugins/workspace-plugin/src/theia-commands.ts

```diff
@@ -203,8 +204,12 @@ export class TheiaImportZipCommand implements TheiaImportCommand {
         const importZip = async (progress: theia.Progress<{ message?: string; increment?: number }>, token: theia.CancellationToken): Promise<void> => {
             try {
                 // download
-                const wgetArgs = [this.locationURI!, '-O', this.zipfilePath];
-                await execute('wget', wgetArgs);
+                const curlArgs = ['-L', '--output', this.zipfilePath];
```

Comment:
should we also add `-s` and `-S` flags?

Reply:
we decided not to add the silent flag, in order to keep the error message on the output in case of failure. Does it make sense to you ?

Reply:
```
-S/--show-error
When used with -s it makes curl show an error message if it fails.
```